### PR TITLE
Code refactoring component

### PR DESCRIPTION
**Pull request by chris-kuehn**

### CODE REVIEW

File: components/Section.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+
+export default function Wrapper({ children }) {
+  return <StudioGeneratorSection>{children}</StudioGeneratorSection>;
+}
```
Comment (F-Kirchhoff) [inline components/Section.js:5]: you can export the component "StudioGeneratorSection" directly. You don't need to create a second Component here called Wrapper. You can do that like this:
```js

const StudioGeneratorSection = styled.div`
  ...styling
`
export default StudioGeneratorSection
```
Keep in mind that a default export can be imported under any name you like, they can be "renamed when imported" so to speak. So it doesn't matter which variable name you choose here, when you import it you can choose a different name.

File: components/SectionHeadline.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import styled, { css } from "styled-components";
+
+export default function SectionHeadline({ headline }) {
+  return <StyledSectionHeadline>{headline}</StyledSectionHeadline>;
+}
+
+const StyledSectionHeadline = styled.h2`
+  margin-bottom: 0.5rem;
+`;
```
Comment (F-Kirchhoff) [inline components/SectionHeadline.js:9]: Here its the same for the headline (but you must write the headline text between the Component tags like this: `<SectionHeadline>This is the text</SectionHeadline>`

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Chirstian,
this looks already pretty good. I like how you use the hsl colors with the variables. Please check my suggestions, you can remove some unnecessary code.

**Comment by chris-kuehn** [comment]
Thank you for the feedback and the explinations! :)

I implemented everything :)

**Pull request merged**